### DISCUSSION TITLE
Drag and Drop wanted

### DISCUSSION THREAD

**Discussion by restructor1** [standardnotes/forum#2699] [Suggestions]
Hi.
It should be best, if a user can drag the tag to note.
Thanks

**Comment by JaspalSuri** [comment]
Hi @restructor1, may I ask why it would be faster to drag a tag to a note instead of typing in multiple tags and using the Autocomplete Tags extension? 

This would be difficult to achieve over time as the list of accumulated tags may expand beyond the window's size.

**Comment by restructor1** [comment]
I understand that tags quantity grow. But according to Evernote experience it would be best and simple to drag and drop tags to note. And possibility to drag the note to tag also.

**Comment by JaspalSuri** [comment]
Thanks for that insight. I'll add it to the feature request!

**Comment by restructor1** [comment]
Thanks

**Comment by cyb3rz3us** [comment]
I actually see a suite of features here that would significantly increase the usability of SN:

1. The feature outlined here --- drag a TAG to a NOTE.
2. Drag a NOTE to a TAG.
3. Multi-select of either TAGs or NOTEs and then be able to drag that selection to the counterpart (i.e. multi-NOTE to a TAG **or** multi-TAG to a NOTE).

I personally find the last feature to really kick up SN's octane so to speak...just MHO...

**Comment by JaspalSuri** [comment]
@zsoltszilvai, what are your thoughts on this? 

(I'm going to put aside what I know is currently planned. :)

**Comment by fineseasalt** [comment]
@restructor1 @cyb3rz3us thank you for your feedback and sharing your ideas. I love them! I've been already exploring the drag & drop functionality in SN a bit, but I'll look into it in more detail in the coming weeks. I took note of your ideas and will see what we can do.

As for selecting multiple notes/tags, that's something that is already discovered and is in the design specs, which is very promising but let's see if/when we can implement it.

FYI: We're working on SN v4 that is going to bring some huge improvements!

Thanks again, and please do let us know if you have more feedback 🙏

**Comment by restructor1** [comment]
> @restructor1 @cyb3rz3us thank you for your feedback and sharing your ideas. I love them! I've been already exploring the drag & drop functionality in SN a bit, but I'll look into it in more detail in the coming weeks. I took note of your ideas and will see what we can do.
> 
> As for selecting multiple notes/tags, that's something that is already discovered and is in the design specs, which is very promising but let's see if/when we can implement it.
> 
> FYI: We're working on SN v4 that is going to bring some huge improvements!
> 
> Thanks again, and please do let us know if you have more feedback 🙏

Thanks. SN may be best.

P.S Hotkeys to delete and to archive needed also.

**Comment by fineseasalt** [comment]
> P.S Hotkeys to delete and to archive needed also.

It's funny you mention this @restructor1 , I'm exploring how we could step our shortcuts game up at this moment :)

You might want to take a look at related ideas [here](https://github.com/standardnotes/forum/issues/973) and [here](https://github.com/standardnotes/desktop/pull/356).

**Comment by restructor1** [comment]
> It's funny you mention this @restructor1 , I'm exploring how we could step our shortcuts game up at this moment :)

Oh. It is nice )))

**Comment by n-kort** [comment]
Drag to Archive or Trash would also be great, as well as multi-select!

**Comment by effieeee** [comment]
Related: https://github.com/standardnotes/forum/issues/949
